pageserver/src/layered_repository.rs

```diff
@@ -251,7 +263,18 @@ impl LayeredRepository {
                 )?;
 
                 // List the layers on disk, and load them into the layer map
-                timeline.load_layer_map(disk_consistent_lsn)?;
+                let loaded_map = timeline.load_layer_map(disk_consistent_lsn)?;
```

Comment:
I think the metadata is loaded twice (see just above this call + inside load_layer_map).
Also disk_consistent_lsn comes from the metadata, seems odd to use it and load it as well

Reply:
Oh my, how did I miss that. Thanks for spotting!